### PR TITLE
➕Propose definition of ableism

### PR DESCRIPTION
**Pull request by dengeist**
> A system that places value on people’s bodies and minds based on societally constructed ideas of normalcy, intelligence and excellence. These constructed ideas of normalcy, intelligence and excellence are deeply rooted in anti-Blackness, eugenics and capitalism.

— T.A. Lewis ([blog source for context](https://www.talilalewis.com/blog/longmore-lecture-context-clarity-grounding))

I like this definition because it acknowledges that ableism cannot be separated from capitalism and anti-Blackness. Crucially, Lewis asks that anyone who cites her note that this is a _working definition_. To respect this, maybe we can work together to crystallize a definition for SelfDefined, and offer credit to Lewis. 

Lewis's blog post and lecture (linked in the blog post) explore the ties between ableism, anti-Blackness, and capitalism, proposing a framework in which ableism cannot exist without racism. This understanding is important, but I'm not sure how best to summarize it on SelfDefined.

As a point of discussion: Lewis says that one does not need to be Disabled to experience ableism. I think this is true, but personally oppose making this qualifier on SelfDefined because I don't want to muddle the conversation about who is most victimized by ableism.

Welcome your thoughts, of course.

### CODE REVIEW

File: _site/index.html [hunk 1]
```diff
@@ -346,6 +346,37 @@ <h3>T-Z</h3>
...
+          </p>
+          <blockquote class="word__definition">
+            A system that places value on people’s bodies and minds based on societally constructed ideas of normalcy, intelligence and excellence. These constructed ideas of normalcy, intelligence and excellence are deeply rooted in anti-Blackness, eugenics and capitalism.
+          </blockquote>
+          <cite>— Talia "TL" Lewis, a working definition</cite>
+          <div class="block__type">
+            <p class="word__type">
+              Impact
+            </p>
+            <p class="word__breakdown">This form of systemic oppression leads to people and society determining who is valuable or worthy based on people’s appearance and/or their ability to satisfactorily produce, excel and “behave.” Importantly, you do not have to be disabled to experience ableism.</p>
```
Comment (tatianamac) [inline _site/index.html]: I have a pet peeve of starting sentences with "This." Instead, I'd suggest:

> As a form of oppression, ableism leads to people and society determining..."

And, I'd like to append to the end "Disabled people can also reinforce ableism internally (manifesting as harm to themselves) and externally (harm to other disabled people).

File: _site/index.html [hunk 2]
```diff
@@ -346,6 +346,37 @@ <h3>T-Z</h3>
...
+            Tool of Oppression
+          </p>
+          <p class="word__title" id="ableism">
+            ableism
+            <span class="word__speech">
+              noun
+            </span>
+          </p>
+          <blockquote class="word__definition">
+            A system that places value on people’s bodies and minds based on societally constructed ideas of normalcy, intelligence and excellence. These constructed ideas of normalcy, intelligence and excellence are deeply rooted in anti-Blackness, eugenics and capitalism.
```
Comment (tatianamac) [inline _site/index.html]: >a system of oppression that: places value on people’s bodies and minds based on socially constructed ideas of normalcy, intelligence, and excellence—ideas founded from and deeply rooted in anti-Blackness, eugenics and capitalism; reinforces systemic power of non-disabled people over disabled people.

[I'm a lowercase wench who believes that we should start definitions in lowercase bc they're a continuation of the ableism, noun, a ... construction FIGHT ME]

File: _site/index.html [hunk 3]
```diff
@@ -346,6 +346,37 @@ <h3>T-Z</h3>
...
+            </p>
+            <p class="word__breakdown">This form of systemic oppression leads to people and society determining who is valuable or worthy based on people’s appearance and/or their ability to satisfactorily produce, excel and “behave.” Importantly, you do not have to be disabled to experience ableism.</p>
+          </div>
+          <div class="block__type">
+            <p class="word__type">
+              Further Reading
+            </p>
+            <p class="word__breakdown">
+              <span class="word__alt">
+                <a class= "word__link" href="https://www.talilalewis.com/blog/longmore-lecture-context-clarity-grounding">Longmore Lecture: Context, Clarity and Grounding.</a>;
```
Comment (tatianamac) [inline _site/index.html]: `                <a class="word__link" rel="noreferrer" href="https://www.talilalewis.com/blog/longmore-lecture-context-clarity-grounding">Longmore Lecture: Context, Clarity and Grounding</a>.`

**Comment by tatianamac** [comment]
@dengeist 👋🏽 Would you still like to contribute to this definition? If no, I can see if someone else would like to finish it out! (I'm eagerly trying to clear out PRs.)

**Comment by dengeist** [comment]
@tatianamac  I would love to finish this contribution! I tried to be fancy and pull in all the changes I have missed, but I've fucked it up. I'll close this PR in favor of a new, cleaner one.

**Comment by tatianamac** [comment]
@dengeist Ah! And I should mention that with the new format the term should be in a markdown file instead, which hopefully is a bit less convoluted:
https://www.selfdefined.app/documentation/

THANK YOU! 💙

**Pull request closed without merging**